README.rst

```diff
@@ -80,7 +80,7 @@ shockingly simple if you know a little Python.
 Install
 -------
 
-You can install beets by typing ``pip install beets``.
+You can install beets by typing ``pip install beets`` or directly from Github (see details [here](https://beets.readthedocs.io/en/stable/faq.html#run-the-latest-source-version-of-beets)).
```

Comment:
Point to latest docs instead of stable docs here, otherwise your change will not be shown.

We currently have stable as the default for beets.readthedocs.io, that is IMO another problem / thing to be discussed currently.